lib/cli/interactive-setup/console-setup-iam-role.js

```diff
@@ -100,10 +100,13 @@ module.exports = {
       await awsRequest(context, cloudFormationServiceConfig, 'describeStacks', {
         StackName: iamRoleStackName,
       });
-      log.warning(
+      log.error(
         'Cannot integrate with Serverless Console: ' +
-          'AWS account is already integrated with another org'
+          'AWS account is already integrated with another org. ' +
+          'You can use the --aws-profile option to use a different AWS Profile.'
```

Comment:
Actually (sorry just realized), `--aws-profile` param is scheduled for deprecation (https://github.com/serverless/serverless/issues/8356) it might be more reliable to advise users to rely on `AWS_PROFILE` env variable instead (?)